tests/testthat/test-fct_lump.R

```diff
@@ -2,44 +2,76 @@ context("fct_lump")
 
 test_that("positive values keeps most commmon", {
   f <- c("a", "a", "a", "b", "b", "c", "d", "e", "f", "g")
+  w <- c( 1,   1,   1,   1,   1,   4,   2,   1,   1,   1) # Sum = 14
```

Comment:
I think the tests would be clearer if you tested weighting in its own block. I think you only need a couple of tests to verify that weighted and the equivalent unweighted return the same values. The code is quite simple so it doesn't need extensive tests.

You might want to add one other test that checks that bad weights generate error messages.